apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -37,7 +37,8 @@
  * Enable PerformanceHints (ADPF).
  */
 public class DynamicWorkloadActivity extends TestOutputActivityBase {
-    private static final double WORKLOAD_MAX = 500.0;
+    private static final int WORKLOAD_HIGH_MIN = 30;
+    private static final int WORKLOAD_HIGH_MAX = 150;
     public static final double LOAD_RECOVERY_HIGH = 1.0;
```

Comment:
Should LOAD_RECOVERY_HIGH and LOAD_RECOVERY_LOW be so close to each other in terms of values?

Reply:
Yes, for hysteresis. I added comments.